CHANGELOG.md

```diff
@@ -40,6 +40,7 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
  * Fixed panic when parsing invalid conflict markers of a particular form.
    ([#2611](https://github.com/martinvonz/jj/pull/2611))
+ * Fixed edit command sometimes not adding the working copy commit into visible heads.
```

Comment:
I think we can describe more exactly when it happens without this becoming too verbose. Something like this:

> Editing a hidden commit now makes it visible

Do you think that's clear enough?

Reply:
Ah, that's much better. Done.